### PR TITLE
login and logout

### PR DESCRIPTION
**Pull request by NT-NhatAnh**
## Related Tickets

- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51472)

## WHAT (optional)

- login and logout feature

## HOW

- create session controlller and login view

### CODE REVIEW

**Comment by NT-NhatAnh** [comment]
ready

**Review by ghost — COMMENTED** [review]
chưa pass ci

**Comment by NT-NhatAnh** [comment]
ready

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (ngocvt-0484) [inline app/helpers/static_pages_helper.rb]: bỏ file trống nhé

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            </div>
+            <div class="form-group text-center">
+              <%= f.submit t(".login"), class: "btn btn-primary" %>
+            </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div id="colorlib-subscribe" style="background-image: url(<%= asset_path("img_bg_2.jpg") %>);" data-stellar-background-ratio="0.5">
```
Comment (ngocvt-0484) [inline app/views/sessions/new.html.erb]: k viết css ở view nhé e
cho vào file css

**Comment by NT-NhatAnh** [comment]
ready

**Pull request merged**